lib/components/form/list.vue

```diff
@@ -114,13 +113,11 @@ export default {
       const s = form.submissions !== 1 ? 's' : '';
       return `${count} submission${s}`;
     },
-    updatedAt(form) {
-      const updatedAt = form.updatedAt != null ? form.updatedAt : form.createdAt;
-      return moment(updatedAt).fromNow();
+    lastModified(form) {
+      return formatDate(form.updatedAt != null ? form.updatedAt : form.createdAt);
```

Comment:
can you not use the new `updatedOrCreatedAt` here?

Reply:
I haven't added the form presenter to the `FormList` component yet, but once I do, I'll start using `updatedOrCreatedAt` and can probably simplify the code in other places as well. I'm thinking I'll add the presenter the next time I touch the component.